pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -530,19 +530,20 @@ func updateObjectsLabels(crLables map[string]string, objs []*unstructured.Unstru
 		// Label objects with version of the operator they were created by
 		labels[cnaov1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
 		labels[names.PROMETHEUS_LABEL_KEY] = names.PROMETHEUS_LABEL_VALUE
+		labels[names.MANAGED_BY_LABEL_KEY] = names.MANAGED_BY_LABEL_DEFAULT_VALUE
 		err = updateObjectTemplateLabels(obj, labels, []string{names.PROMETHEUS_LABEL_KEY})
```

Comment:
we might want to consider please refactor the double call of `updateObjectTemplateLabels`